spec/custom/index.html

```diff
@@ -176,7 +176,7 @@ <h4 id="custom-elements-custom-tag-example">Creating a custom tag</h4>
 <p>We then need to use this class to define the element:</p>
 
 <pre class="highlight">
-document.defineElement("flag-icon", FlagIcon);</pre>
+document.registerElement("flag-icon", FlagIcon);</pre>
```

Comment:
Yes, you're right! [This interface](http://w3c.github.io/webcomponents/spec/custom/#custom-elements-api) should be used.

Anyway. Will be great if other developers will read an example that works. The introduction __section is non-normative__ after all. So, today we read an example that works and allows us to understand the meaning of custom components.